dist/keysim.js

```diff
@@ -1,123 +1,110 @@
-(function (global, factory) {
```

Comment:
The browserify boilerplate is a bit overkill. Also, I think we should stop checking `dist/` into git.